opencmp/entry_points.py

```diff
@@ -31,12 +25,7 @@ def run_opencmp():
 
     elif len(sys.argv) == 2: # if user did provide a configuration path
         config_file_path = sys.argv[1]
-        config_parser = ConfigParser(config_file_path)
-
-    elif len(sys.argv) == 3: # if the user provided both config path and optional config_parser argument
-        config_file_path = sys.argv[1]
-        config_parser = sys.argv[2]
-        config_parser = cast(ConfigParser, config_parser)    
+        config_parser = ConfigParser(config_file_path)   
```

Comment:
Don't have to do this. Just run it without passing the ConfigParser. `run()` does this line right at the beginning.